src/runtime/smart-button-api.js

```diff
@@ -107,6 +108,10 @@ export class SmartSubscriptionButtonApi {
       'width': '100%',
     });
     this.button_.appendChild(this.iframe_);
+    if (this.deps_.config().analyticsMode == AnalyticsMode.IMPRESSIONS) {
```

Comment:
may not need this. sending context is not so controversial.

Reply:
Let's confirm with John and Shams.

Reply:
Resolved this offline. Removing if statement.